src/voter/voting_round.rs

```diff
@@ -343,6 +343,7 @@ impl<H, N, E: Environment<H, N>> VotingRound<H, N, E> where
 							}
 						}
 						Err(crate::Error::NotDescendent) => last_round_estimate.0,
+						Err(crate::Error::MultipleForksSupportPredicate) => { return Err(crate::Error::MultipleForksSupportPredicate.into()) },
```

Comment:
```suggestion
						Err(e) => { return Err(e.into()); },
```

Reply:
is this a good thing to do? when a new error will be added it would silently wildcard here and the exhausive pattern checking would be of no help